react/components/Link.tsx

```diff
@@ -23,6 +23,7 @@ interface Props extends NavigateOptions {
   className?: string
   target?: string
   waitToPrefetch?: number
+  rel?: HTMLAnchorElement['rel']
```

Comment:
What do you think about `props` extends HTMLAnchorElement?

Reply:
Are you suggesting to make `props` extend both NavigateOptions _and_ HTMLAnchorElement? I didn't know that in TS you could extend two interfaces simultaneously

Reply:
Yeah, you can extend how many as you want to. Take a look [here](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAhgezcg3gXwFCiSyIoBGcU2+h08SyAHqWJhA5CACYDOqGANMnKVcQA)

Reply:
![immagine](https://user-images.githubusercontent.com/35635227/127683972-fccb8247-9ca3-40a8-adcd-61b00e270d98.png)
Am I missing something?

Reply:
You're seeing this because apparently, this `hash` property exists in both of them.

We can merge the PR as is just so it doesn't get held up